src/main/java/org/openstreetmap/atlas/geography/atlas/change/FeatureChange.java

```diff
@@ -75,6 +76,12 @@ public static FeatureChange remove(final AtlasEntity reference)
     }
 
     public FeatureChange(final ChangeType changeType, final AtlasEntity reference)
+    {
+        this(changeType, reference, null);
```

Comment:
IMO null should be avoided in favor of an empty String, but this is probably a nit. thoughts?

Reply:
You're probably right. But this is also going to be removed once the bug is solved. Just temporarily added this for logging purposes. Although I suppose we can make a case to @matthieun to add this functionality permanently. However, it would need to respect the merge semantics in a sensible way.

Reply:
fair enough!